synapse/handlers/event_auth.py

```diff
@@ -49,19 +49,16 @@ def __init__(self, hs: "HomeServer"):
         self._store = hs.get_datastore()
         self._server_name = hs.hostname
 
-    async def check_from_context(
+    async def check_auth_rules_from_context(
         self,
-        room_version: str,
+        room_version_obj: RoomVersion,
         event: EventBase,
         context: EventContext,
```

Comment:
the eagle-eyed reviewer will notice that `context` is unused, so may wonder why I'm not getting rid of it, and why the function is still called `..._from_context`.

The answer is that I'm planning to add some extra tests based on the context, in the near future.